superset-frontend/src/views/CRUD/welcome/Welcome.tsx

```diff
@@ -106,9 +116,17 @@ const WelcomeNav = styled.div`
 `;
 
 function Welcome({ user, addDangerToast }: WelcomeProps) {
+  const userid = user.userId;
+  const id = userid.toString();
   const recent = `/superset/recent_activity/${user.userId}/?limit=6`;
```

Comment:
Is it worth reconsidering this limit, now that we show more things?

Reply:
I'm not sure. This seem to be an issue for people when fetching recents so I adding more here might cause some performance issues.